pages/typography/typography.html

```diff
@@ -0,0 +1,93 @@
+        </article>
+        <article class="section-content underline">
+          <h2>Headings</h2>
+          <p class="small-text">All HTML headings h1 to h5 are available </p>
+          <div class='example-code'>
+            <h1 class="underline">h1 starUI Heading</h1> 
+            <h2 class="underline">h2 starUI Heading</h2> 
+            <h3 class="underline">h3 starUI Heading</h3> 
+            <h4 class="underline">h4 starUI Heading</h4> 
+            <h5 >h5 starUI Heading</h5> 
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=1000&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Ch1%253Eh1%2520starUI%2520Heading%253C%252Fh1%253E%2520%250A%253Ch2%253Eh2%2520starUI%2520Heading%253C%252Fh2%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%253Ch3%253Eh3%2520starUI%2520Heading%253C%252Fh3%253E%2520%250A%253Ch4%253Eh4%2520starUI%2520Heading%253C%252Fh4%253E%2520%250A%253Ch5%253Eh5%2520starUI%2520Heading%253C%252Fh5%253E%2520"
+              style=" height: 279px; border:0; transform: scale(1); overflow:hidden;"
+              sandbox="allow-scripts allow-same-origin">
+            </iframe>
+          </div>
+        </article>
+        <article class="section-content underline">
+          <h2>Text</h2>
+          <p class="small-text">There are 5 pre-defined classes for text ,they are as follows</p>
+          <div class='example-code'>
+            <p class='x-large-text underline'>This is extra large text</p>
+            <p class='large-text underline'>This is large text</p>
+            <p class='medium-text underline'>This is medium text</p>
+            <p class='small-text underline'>This is small text</p>
+            <p class='x-small-text '>This is extra small text</p>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=1000&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cp%2520class%253D%27x-large-text%27%253EThis%2520is%2520extra%2520large%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27large-text%27%253EThis%2520is%2520large%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27medium-text%27%253EThis%2520is%2520medium%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27small-text%27%253EThis%2520is%2520small%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27x-small-text%27%253EThis%2520is%2520extra%2520small%2520text%253C%252Fp%253E"
+              style=" height: 279px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
avoid inline styling for iframe

Reply:
> avoid inline styling for iframe

yea this was the default inline style provided by carbonsh